src/oslogin_utils.cc

```diff
@@ -776,19 +791,18 @@ std::vector<string> ParseJsonToSshKeysSk(const string& json) {
 
 bool ParseJsonToPasswd(const string& json, struct passwd* result, BufferManager*
                        buf, int* errnop) {
+  bool ret = false;
```

Comment:
you're moving it up here but only starting to use it where we were before, did you mean to use if as return on the `root == NULL` check? like you did down below?

Reply:
In general, though, I see the use of `ret` return variables as a bit of a wart, and something to be avoided wherever possible. 

So, even moreso than consistent styling of the `ret` variable, I would like to see return variables simply disused in this file. :P